contentcuration/contentcuration/frontend/channelEdit/views/trash/TrashModal.vue

```diff
@@ -1,173 +1,170 @@
 <template>
 
-  <div>
```

Comment:
Let's revert this. And move the `<MoveModal...>` code to be a child of the `div` as previously. That way we keep the pr focused to the confirm dialog only. This will prevent potential regressions arising from moving around the code. 
```vue
<template>
    <div>
        <FullscreenModal>
            ......
        </FullscreenModal>
        <MoveModal
            v-else
            .....
        />
    </div>
</template>
```